Kitodo/src/test/java/org/kitodo/production/helper/HelperComparatorTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper;
+
+import java.util.Map;
+import java.util.TreeSet;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HelperComparatorTest {
+    @Test
+    public void testCompareLists() {
```

Comment:
_please test edges:_

empty left list
empty right list
both lists empty
both lists identical
duplex entries
no entries which are in both lists
lists with different sizes
...

Reply:
Input is TreeSet. Duplex entries is not possible.